raiden/tests/integration/rpc/test_assumptions.py

```diff
@@ -131,11 +133,11 @@ def test_duplicated_transaction_same_gas_price_raises(deploy_client):
         contract_proxy.contract_address,
     )
 
-    gas = contract_proxy.estimate_gas('ret') * 2
+    startgas = contract_proxy.estimate_gas('ret') * 2
```

Comment:
I don't understand the need for doubling the estimate here and below. Either don't double it, or use the safety `GAS_FACTOR` from `raiden.constants`, if this is what you want to test, instead.

Reply:
That is a good question, I looked in the commit history and this `* 2` factor was introduced with the test itself, maybe it was because of the wrong estimation when `revert` was introduced?

Reply:
this test is about the gas price, not the gas limit.